app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsActionFragment.kt

```diff
@@ -80,6 +99,15 @@ class TransactionDetailsActionFragment : BaseViewBindingFragment<FragmentTransac
         }
     }
 
+    private fun getTransferItem(address: Solidity.Address, amount: String): TxTransferActionView {
+        val item = TxTransferActionView(requireContext())
+        val layoutParams = LayoutParams(MATCH_PARENT, WRAP_CONTENT)
+        layoutParams.setMargins(0, dpToPx(16), 0, -dpToPx(8))
```

Comment:
negative margin? seems a bit hacky, what are you trying to do with that?

Reply:
TxTransferAction has an internal margin that is too big for decoded action screen.